pom.xml

```diff
@@ -4,7 +4,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>1.401</version>
+        <version>2.21</version>
     </parent>
 
     <artifactId>versioncolumn</artifactId>
```

Comment:
The name below should be updated IMHO. The original one was not good, with newer functionality it's getting worse